SCOPE.md

```diff
@@ -0,0 +1,53 @@
+# Scope
+
+The REST API specification is meant to provide the functionality currently offer by Wikibase "Action API".
+
+The scope of the schema is limited to the primary entity types (Items & Properties) and the main [CRU(D)](https://en.wikipedia.org/wiki/Create,_read,_update_and_delete) actions for those entities.
+
+## Intentionally not implemented
+
+The following elements are considered non essential for the design decisions and are intentionally excluded from the prototype schema:
+
+### Deletions
+
+As entity specific deletion is not yet covered by any Wikibase API the deletion part of CRUD is not covered in this specification.
```

Comment:
It [is in the spec, though](https://github.com/wmde/wikibase-rest-fantasies/blob/54d59b9/paths/entities/single.json#L44). Or does this just mean people should not think about it too much?

Reply:
That is a good question and probably something we want to discuss and clarify.
This is technically a "new" API, but is part of the core CRUD we are aiming for.

I'd be pro just removing this section.

Reply:
Took me a moment but now remembered that this is functionality currently really not exposed through the action API at all.

I don't have a strong opinion if we should leave it (because we worked on it already, it would be useful for completeness sake in the end) or remove it (because it is not directly useful for gathering feedback about people's ability to cover their current use cases). Removing it (from the spec) would be closer to the agile spirit, I guess.